Makefile

```diff
@@ -66,7 +68,8 @@ EPOCH_TEST_COMMIT ?= e68e0e1110e64f906f9b482e548f17d73e02e6b1
 .gitvalidation:
 	@which git-validation > /dev/null 2>/dev/null || (echo "ERROR: git-validation not found. Consider 'make clean && make tools'" && false)
 ifeq ($(TRAVIS),true)
-	@git-validation -q -run DCO,short-subject,dangling-whitespace
+	GIT_CHECK_EXCLUDE=${NO_GIT_VALIDATION} git-validation -q -run DCO,short-subject,dangling-whitespace
```

Comment:
excluding the configuration file from the gitvalidation scan due to the conf file having/and needing/ whitespace